### PR TITLE
fix sql query

### PR DESCRIPTION
**Pull request by furkhat**

### CODE REVIEW

File: monitor/schedule.go [hunk 1]
```diff
@@ -422,6 +412,27 @@ func (m *Monitor) scheduleUpdateCurrentSupply(el *data.EthLog, jobType string) {
...
+				        ON jobs.id=eth_logs.job`,
+		hashHex,
+		data.JobAgentAfterOfferingMsgBCPublish,
+		data.JobClientAfterOfferingMsgBCPublish,
+	)
+
+	if err != nil && err != sql.ErrNoRows {
+		m.logger.Error(err.Error())
+	}
+	if job != nil {
```
Comment (dzeckelev) [inline monitor/schedule.go]: this condition is useless.
may be better `if job.ID != ""`?

**Comment by furkhat** [comment]
@dzeckelev updated.

File: monitor/schedule.go [hunk 2]
```diff
@@ -422,6 +412,23 @@ func (m *Monitor) scheduleUpdateCurrentSupply(el *data.EthLog, jobType string) {
 	}
 }
 
+func (m *Monitor) offeringPublishJobID(hash common.Hash) string {
+	hashHex := fmt.Sprintf("0x%064v", hash.Hex())
+	job := &data.Job{}
+	err := m.db.SelectOneTo(job, `WHERE eth_logs.topics->>2 = $1
```
Comment (ababo) [inline monitor/schedule.go]: Can you decrease the indentation of the SQL-query? I mean:

```
err := m.db.SelectOneTo(`
        WHERE ...`, hashHex, ...
```

**Comment by furkhat** [comment]
@simon updated.

**Pull request merged**